rp2040-hal/src/dma/single_channel.rs

```diff
@@ -31,6 +31,12 @@ pub trait SingleChannel: Sealed {
         }
     }
 
+    /// Check if the DMA_IRQ_0 signal for this channel is enabled.
+    fn is_enabled_irq0(&mut self) -> bool {
+        // Safety: We only use the atomic alias of the register.
```

Comment:
This is also not true. The read is fine, AFAICT, but it's not because of some atomic alias.

Reply:
To be fair this is a nit, you know this particular safety comment isn't ever going to lead to violated safety invariants.
I was going to fix it up port-merge to avoid having to press "approve" again on the CI run because this is a first time contributor.

Reply:
Yes, of course, it doesn't actually affect correctness of the code. But it can be confusing for someone trying to understand the code. Instead of a wrong safety comment, better remove it or write something like TBD.